src/bbb/tasks/classification.py

```diff
@@ -75,6 +78,7 @@ def run_bbb_mnist_classification_evaluation(model_path: str):
     input_dim = 28*28,
     output_dim = 10,
     hidden_units = 1200,
+    hidden_layers = 2,
```

Comment:
@mbronckers Just to be sure you spot this - I've got both the BBB and DNN set to 2 layers for classification, to match the paper